helm/cluster/templates/clusterapi/_helpers_files.tpl

```diff
@@ -0,0 +1,88 @@
+{{- define "cluster.internal.kubeadm.files" }}
```

Comment:
Kubeadm helper templates which are common for both control plane and worker nodes.